DEVELOPING.md

```diff
@@ -41,4 +41,28 @@ make run/docker
 By default, we run with a SQLite configuration.
 You can run other platforms like CockroachDB, MariaDB, PostgreSQL, and MySQL as well.
 
+## Rebuilding the base images
+
+These images are are used to to build the remainder of the application as well as a base
+for the runtime docker images. The default is to build the base images without a registry
+prefix:
```

Comment:
nit: one sentence per line

Very rarely do we use a target width pattern in the project. I've found that it helps me identify run-ons and then I don't need to re-adjust things when there are edits.

Also, your first sentence here has `are are used`. `are used` is passive voice. I'd probably frame it more as `These images provide the foundation for building the rest of the application.`

`s/docker/container/` in text.

Reply:
And this is why I need an editor... one moment and I'll correct this.

Reply:
If I'm reading this right, you're using a sentence per line?

Reply:
Yeah, it's a lot easier to edit later on if something needs to be added to a sentence.